charts/rode-collector-harbor/Chart.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: rode-collector-harbor
+description: A Helm chart for the Rode Harbor Collector. Meant to be deployed alongside Rode
+
```

Can you merge main into this branch so that the linter runs? It will probably ask for the following to be added
```suggestion
maintainers:
  - name: Liatrio
    email: rode@liatrio.com
```
Also lets get rid of all of the `helm create` comments